src/data/reference-data.js

```diff
@@ -68,6 +68,48 @@ export default {
     hapmap_resource_vcf_index: 'gs://gcp-public-data--broad-references/hg38/v0/hapmap_3.3.hg38.vcf.gz.tbi',
     mills_resource_vcf_index: 'gs://gcp-public-data--broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi',
     ref_pac: 'gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.fasta.64.pac'
+  },
+  'Mmul-10': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/M.mulatta/Mmul_10/GCF_003339765.1_Mmul_10_genomic.fna'
+  },
+  'Clint-PTRv2': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/P.troglodytes/Clint_PTRv2/GCF_002880755.1_Clint_PTRv2_genomic.fna'
+  },
+  GRCm39: {
+    ref_fasta: 'gs://gcp-public-data--broad-references/GRCm39/GCF_000001635.27_GRCm39_genomic.fna'
+  },
+  'mRatBN7-2': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/R.norvegicus/mRatBN7.2/GCF_015227675.2_mRatBN7.2_genomic.fna'
+  },
+  'Rnor-6-0': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/R.norvegicus/Rnor_6.0/GCA_000001895.4_Rnor_6.0_genomic.fna'
+  },
+  'Release-6-plus-ISO1-MT': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/D.melanogaster/Release_6_plus_ISO1_MT/GCF_000001215.4_Release_6_plus_ISO1_MT_genomic.fna'
+  },
+  'UCB-Xtro-10-0': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/X.tropicalis/UCB_Xtro_10.0/GCF_000004195.4_UCB_Xtro_10.0_genomic.fna'
+  },
+  GRCz11: {
+    ref_fasta: 'gs://gcp-public-data--broad-references/D.rerio/GRCz11/GCF_000002035.6_GRCz11_genomic.fna'
+  },
+  WBcel235: {
+    ref_fasta: 'gs://gcp-public-data--broad-references/C.elegans/WBcel235/GCF_000002985.6_WBcel235_genomic.fna'
+  },
+  R64: {
+    ref_fasta: 'gs://gcp-public-data--broad-references/S.cerevisiae/R64/GCF_000146045.2_R64_genomic.fna'
+  },
+  'ROS-Cfam-1-0': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/C.lupus_familiaris/ROS_Cfam_1.0/GCF_014441545.1_ROS_Cfam_1.0_genomic.fna'
+  },
+  'UU-Cfam-GSD-1-0': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/C.lupus_familiaris/UU_Cfam_GSD_1.0/GCA_011100685.1_UU_Cfam_GSD_1.0_genomic.fna'
+  },
+  'Sscrofa11-1': {
+    ref_fasta: 'gs://gcp-public-data--broad-references/S.scrofa/Sscrofa11.1/GCF_000003025.6_Sscrofa11.1_genomic.fna'
+  },
+  'ARS-UI-Ramb-v2-0': {
```

Comment:
Unfortunately the true assembly names for these include dots, which are not allow in attribute names. I wonder if it may be worth also including a displayName value, which is what we'd show in the dropdown...

Reply:
Then again, I guess adding that wouldn't change anything here, so it could be done later.